metadata-ingestion/src/datahub/ingestion/graph/client.py

```diff
@@ -543,6 +547,117 @@ def get_container_urns_by_filter(
             logger.debug(f"yielding {x['entity']}")
             yield x["entity"]
 
+    def _bulk_fetch_schema_info_by_filter(
+        self,
+        *,
+        platform: Optional[str] = None,
+        platform_instance: Optional[str] = None,
+        env: Optional[str] = None,
+        query: Optional[str] = None,
+        container: Optional[str] = None,
+        status: RemovedStatusFilter = RemovedStatusFilter.NOT_SOFT_DELETED,
+        batch_size: int = 100,
+        extraFilters: Optional[List[SearchFilterRule]] = None,
+    ) -> Iterable[Tuple[str, "SchemaInfo"]]:
+        """Fetch schema info for datasets that match all of the given filters.
+
+        Filters are combined conjunctively. If multiple filters are specified, the results will match all of them.
```

Comment:
given that this comment is already on get_urns_by_filter, we don't need to copy it here

Reply:
removed this, as this is a private method. I would have preferred to keep the docstrings for any public methods.